packages/apps/explorer/src/constants/search.ts

```diff
@@ -1,4 +1,4 @@
-import type { ISearchItem } from '@/components/search/search';
+import type { ISearchItem } from '@/components/search-component/search-component';
```

Comment:
maybe make it '@/components/search/search-component/search-component'
and have all the search components close to eachother?

im adding '@/components/search/search-results/search-results'